src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -315,14 +317,21 @@ $(function () {
   };
 
   // Update href in navbar
-  function updateHref() {
-    var toc = $('#sidetoc');
-    var breadcrumb = new Breadcrumb();
-    loadNavbar();
-    loadToc();
+  function renderNavbar() {
+    var navbar = $('#navbar ul')[0];
+    if (typeof (navbar) === 'undefined') {
+      loadNavbar();
+    } else {
+      $('#navbar ul a.active').parents('li').addClass(active);
+      renderBreadcrumb();
```

Comment:
yes, each time breadcrumb contains potential changes, we re-generate breadcrumb, because both navbar and sidebar are load asynchronously, the order is not guaranteed. 

---
In reply to: [108879208](https://github.com/dotnet/docfx/pull/1509#discussion_r108879208) [](ancestors = 108879208)